### PR TITLE
Submit Pull

### PR DESCRIPTION
**Pull request by Oklukeok**
new Loom Video: https://www.loom.com/share/c4de6be58eb947e0be756504e2d347a3

### **_Features:_**
Welcome to CashTrack App, this app was made for the Ruby on rails capstone.

**_### Made with:_**
- Ruby on rails
- Devise
- PostgreSql

### **_Deploy:_**
- render.com

### **_Functionalities:_**

- Create your own user 
- Log in and Log Out of your account
- Create customs categories
- Create customs transactions
- Sum the total amounts of outgoings
- Sum the partial amount of outgoings by category

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,123 +99,56 @@ After you're finished please remove all the comments and instructions!
...
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **You can create your own user account and track your personal finances**
+- **You can create your customs categories to separate your outgoings**
+- **You can add every transaction you did**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
```
Comment (brytebee) [inline README.md:108]: - [x] **Required:** You have done great work so far. However, the presentation video has no audio. Kindly update this.
- [x] Your acknowledgement section is required to contain credits to the owner of the original design, [Gregoire Vella on Behance](https://www.behance.net/gregoirevella). Kindly update it.

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+#landing h1 {
+  color: white;
+  font-family: 'Pacifico', cursive;
+  font-size: 4em;
+}
+
+#landing a {
+  color: white;
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
```
Comment (brytebee) [inline app/assets/stylesheets/application.css]: Great designs so far. 👏 

Kindly note the required font families for the headers and body of your design.

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Oklukeok👋, 

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️
- Users can create an account and can also log into the application.✔️
- The project is hosted.✔️
- Great use of migration and indexes are properly used.✔️
- Test passed and all green.✔️


##

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification. 
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @Oklukeok 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Your app's UI design is a far cry from how it should look based on the Behance design. Please consider following the design guidelines to a tee in order to gain experience on how to follow specifications and also have a more beautiful project 👍 
## Your Design
image

## Design Guideline
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**